README.md

```diff
@@ -282,7 +161,3 @@ I would like to thank...
 ## 📝 License <a name="license"></a>
 
 This project is [MIT](./LICENSE) licensed.
```

Comment:
[REQUIRED] This time we are being taken to a 404 page. I can see that you were probably trying to follow my suggestion but I can also notice that you don't have a license file in the root of this project. You can check this [link](https://blog.avneesh.tech/how-to-add-a-license-to-a-github-repository) out for some guidance or make a google search for better understanding